lua/hurl/utils.lua

```diff
@@ -312,5 +312,80 @@ util.has_file_in_opts = function(opts)
 
   return false
 end
+-- Function to extract the URL from the .hurl file
```

Comment:
If as a concept the PR is liked and this is an issue, I can improve that. It works for single line urls currently, but it is right that it might have an issue if you split (if somebody splits the url ofc).

Let me know

Reply:
Hi @Xouzoura Thank you for your PR. Yeah, I like your idea on this. Could you please check and fix the review bots when you're available? 🙏

Reply:
@jellydn Cool! Had no time so far, I will try this weekend.